test/functional/ui/spell_spec.lua

```diff
@@ -9,31 +9,33 @@ local feed_command = helpers.feed_command
 
 describe('combining long lines', function()
   local screen
+
   before_each(function()
       clear()
-      screen = Screen.new(264, 4)
+      screen = Screen.new(80, 4)
       screen:attach()
-      screen:set_default_attr_ids()
   end)
 
   after_each(function()
     screen:detach()
   end)
 
   it('#7937 successfully joins lines with "set spell"', function()
-    feed_command('set spell nowrap')
+    feed_command('set nowrap spell')
```

Comment:
Doesn't it crash also with `wrap` (the default)? I don't we should change default values if not necessary.

Reply:
It does.  Updating.